reference/docs-conceptual/whats-new/experimental-features.md

```diff
@@ -0,0 +1,215 @@
+---
+ms.date: 04/28/2020
+title: Using experimental features in PowerShell
+description: Lists the currently available experimental features and how to use them.
+---
+# Using experimental features in PowerShell
+
+The Experimental Features support in PowerShell provides a mechanism for experimental features to
```

Comment:
Should this experimental features be proper-cased or should it be lower-cased like all the others?  If lower, should the TOC be lower?  If upper, should the Title and H1 be upper and following uses?

Reply:
Fixed